run/opencl/opencl_misc.h

```diff
@@ -34,6 +34,10 @@ typedef uint64_t host_size_t;
 typedef uint32_t host_size_t;
 #endif
 
+#ifndef NULL
+#define NULL ((void*)0)
+#endif
```

Comment:
Why do we need to define `NULL`, and why didn't we need it before?

Reply:
I found it in `encfs_kernel.cl` and was surprised it's not defined in OpenCL. I just thought we should move it to `opencl_misc.h` before more files start defining it locally.